schema/deploy/shipping/views.sql

```diff
@@ -728,4 +728,53 @@ comment on view shipping.return_results_v2 is
     'Version 2 of view of barcodes and presence/absence results for return of results on website';
 
 
+create or replace view shipping.hcov19_observation_with_presence_absence_result_v1 as
+
+select
+        presence_absence.created::date as sample_tested_date,
+        present as test_result,
+        sample.details->>'sample_origin' as sample_manifest_origin,
+        best_available_encounter_date,
+        age_range_fine,
+        sex,
+        residence_puma,
+        best_available_site,
+        individual,
+        sample.identifier as sample,
+        symptoms,
+        best_available_site_type
+
+    from shipping.sample_with_best_available_encounter_data_v1
+    join warehouse.sample using (sample_id)
+    full outer join shipping.incidence_model_observation_v3 on
+      (sample.identifier = incidence_model_observation_v3.sample)
```

Comment:
What columns ultimately come from the `incidence_model_observation_v3` view joined here? I'd guess it would be cleaner to grab them from core fact tables than re-use that view. Too many cross-view dependencies.

Reply:
Not too many columns. I thought it would make for cleaner `age_range` selection, but I agree that reducing cross-view dependencies is probably a good thing.

Reply:
The list of columns from `incidence_model_observation_v3` is:
* `age_range_fine`
* `age_range_fine_upper`
* `age_range_fine_upper`
* `sex`
* `symptoms`